recipe/activate-clang.sh

```diff
@@ -119,13 +119,22 @@ if [ "${CONDA_BUILD_SYSROOT:-0}" != "0" ] && [ "${CONDA_BUILD_STATE:-0}" = "TEST
   unset CONDA_BUILD_SYSROOT
 fi
 
-CONDA_BUILD_SYSROOT_TEMP=${CONDA_BUILD_SYSROOT:-${SDKROOT:-0}}
-if [ "${CONDA_BUILD_SYSROOT_TEMP}" = "0" ]; then
+if [ "${CONDA_BUILD:-0}" = "0" ]; then
+  # outside of conda build we do not purposefully use CONDA_BUILD_SYSROOT
   if [ "${SDKROOT:-0}" = "0" ]; then
     CONDA_BUILD_SYSROOT_TEMP=$(xcrun --show-sdk-path)
   else
     CONDA_BUILD_SYSROOT_TEMP=${SDKROOT}
   fi
+else
+  CONDA_BUILD_SYSROOT_TEMP=${CONDA_BUILD_SYSROOT:-${SDKROOT:-0}}
+  if [ "${CONDA_BUILD_SYSROOT_TEMP}" = "0" ]; then
+    if [ "${SDKROOT:-0}" = "0" ]; then
+      CONDA_BUILD_SYSROOT_TEMP=$(xcrun --show-sdk-path)
+    else
+      CONDA_BUILD_SYSROOT_TEMP=${SDKROOT}
+    fi
+  fi
 fi
```

Comment:
Can you remove this change? We should support `CONDA_BUILD_SYSROOT`, but not set it by ourselves.

Reply:
Done!